### PR TITLE
Update example files with new options

### PR DESCRIPTION
**Pull request by mjura**

### CODE REVIEW

File: examples/create-example.yaml [hunk 1]
```diff
@@ -8,8 +8,10 @@ spec:
   clusterName: "my-cluster"
   baseUrl: "https://management.azure.com/"
   authBaseUrl: "https://login.microsoftonline.com"
-  tenantId: "REPLACE_WITH_TENANT_ID"
   azureCredentialSecret: "REPLACE_WITH_K8S_SECRETS_NAME"
+  privateCluster: false,
+  windowsAdminUsername: "my-admin-user",
+  windowsAdminPassword: "my-secure-passworD",
```
Comment (cmurphy) [inline examples/create-example.yaml:14]: If you want to include windows in this example then you can remove https://github.com/rancher/aks-operator/blob/master/examples/create-windows-example.yaml

For windows to work you need to set `networkPlugin`, `virtualNetwork`, and `subnet` and the name of the node pool is restricted to 6 characters, `workers2` is too long. This is why I had broken it out into a separate example.

**Review by thedadams — APPROVED** [review]

**Review by cmurphy — CHANGES_REQUESTED** [review]

File: examples/create-example.yaml (hunk 1 above)
Comment (mjura) [reply examples/create-example.yaml]: updated

**Review by thedadams — DISMISSED** [review]

File: examples/create-example.yaml [hunk 2]
```diff
@@ -8,8 +8,10 @@ spec:
   clusterName: "my-cluster"
   baseUrl: "https://management.azure.com/"
   authBaseUrl: "https://login.microsoftonline.com"
-  tenantId: "REPLACE_WITH_TENANT_ID"
   azureCredentialSecret: "REPLACE_WITH_K8S_SECRETS_NAME"
+  privateCluster: false,
+  linuxAdminUsername: "rancher-user",
+  linuxAdminPassword: "rancher-password",
```
Comment (cmurphy) [inline examples/create-example.yaml:14]: There's no `linuxAdminPassword`, it's `linuxSSHPublicKey`

**Review by cmurphy — CHANGES_REQUESTED** [review]

**Review by rmweir — APPROVED** [review]
LGTM once Colleen's feedback has been addressed.

File: examples/create-example.yaml (hunk 2 above)
Comment (mjura) [reply examples/create-example.yaml]: ahh yeah, you are right, but there is `sshPublicKey` option

**Review by cmurphy — APPROVED** [review]

**Review by rmweir — APPROVED** [review]
LGTM

**Pull request merged**